### ISSUE TITLE
CPU Utilization not working release 2.4

### ISSUE THREAD

**Issue by jackjt8** [chunky-dev/chunky#1047]
Chunky  2.3
4 threads at 100% 
![image](https://user-images.githubusercontent.com/3740917/135923746-f7cb9b5f-6914-4c57-86c3-754f62ab3fba.png)

4 threads at 1%
![image](https://user-images.githubusercontent.com/3740917/135923566-f8a1e381-7a03-4312-a77d-46a729ddba5f.png)

---

Chunky 2.4
4 threads at 100% 
![image](https://user-images.githubusercontent.com/3740917/135922592-5b11923c-7b1c-4cb3-b140-6e415632be58.png)

4 threads at 1%
![image](https://user-images.githubusercontent.com/3740917/135922864-7ec9ef1c-848d-43a5-ac10-67da7e600df5.png)

---

tl;dr Chunky 2.3 CPU Utilization works (as well as it can all things considered) and Chunky 2.4's doesn't. Maybe due to the render manager changes?
